iree/compiler/Dialect/Flow/Transforms/LegalizeConstants.cpp

```diff
@@ -0,0 +1,111 @@
+// Copyright 2019 Google LLC
```

Comment:
How much of this is forked from an existing file? If it's new, the year should be 2021.

Reply:
Updated the header. Bad force of habit to just copy the header :P